Dockerfile

```diff
@@ -51,8 +51,10 @@ RUN \
     mkdir /var/lib/qgis && chown $UID /var/lib/qgis && \
     # Dir for QGIS.ini
     mkdir /etc/QGIS/ && \
-    # Writeable cache directory
+    # Writeable cache directories
     mkdir /.cache && chown $UID /.cache && \
+    # This directory is used to store the wfsprovider cache
+    mkdir /etc/cache && chown $UID /etc/cache && \
```

Comment:
Is this cache path configured in QGIS? It seems a bit unusual that an application would attempt to cache to /etc/cache?

Reply:
The problem is that this cache directory is created in `$QGIS_OPTIONS_PATH/cache`. That means we either have to change the value for `QGIS_OPTIONS_PATH` or create this cache directory ourself, since `www-data` does not have enough permissions to do it.

Reply:
On a side note, `QGIS_OPTIONS_PATH` is hardcoded to `/etc` in the [Dockerfile](https://github.com/qwc-services/qwc-qgis-server/blob/main/qgis3-server.conf.template#L35). This is not really an issue, just important for the understanding.

Reply:
Isn't there a separate env variable to set the cache path? Putting a cache into `/etc` is really disgusting...

Reply:
There is `QGIS_SERVER_CACHE_DIRECTORY` but that only sets the network cache, which does not include the `wfsprovider` cache. `wfsprovider` cache is stored in `QGIS_OPTIONS_PATH/cache`.

Reply:
Could we at least set QGIS_OPTIONS_PATH to `/var/qgis-server` or something? So then we would have

``` 
/var/qgis-server/
/var/qgis-server/cache
/var/qgis-server/QGIS
/var/qgis-server/QGIS/QGIS.INI
[don't know what else comes here]
```

Putting `cache` under `/etc/` makes me want to vomit.

Would moving QGIS_OPTIONS_PATH to `/var/qgis-server` introduce a breaking change on the user or on some other software or scripts that expect some qgis bits and bytes to be under `/etc/`?

Reply:
It could break some users setup if they currently mount a custom qgis.ini to `/etc/QGIS/QGIS3.ini` (which is currently even documented behaviour in README.md).

Reply:
Something like that would work I suppose, though I wonder whether it would not be a better goal to get rid of `/etc` completely and just document the breaking change. At the same time, also pointing `$QGIS_SERVER_CACHE_DIRECTORY` to a more appropriate directory than `/.cache` might be good idea.